etc/afpd/file.c

```diff
@@ -895,7 +895,6 @@ int setfilparams(struct vol *vol,
         }
         switch(  bit ) {
         case FILPBIT_ATTR :
-            change_mdate = 1;
             memcpy(&ashort, buf, sizeof( ashort ));
```

Comment:
Are you sure this is correct? IIrc this breaks AFP 3 conformance.

Reply:
I see, that's good insight. So there's a conflict between AFP3 specs and GS/OS expectations. I think it makes the most sense to discard this patch since the issue it's trying to fix for GS/OS is trivial.